pkg/kubelet/metrics/collectors/resource_metrics_test.go

```diff
@@ -168,6 +172,11 @@ func TestCollectResourceMetrics(t *testing.T) {
 				container_memory_working_set_bytes{container="container_a",namespace="namespace_a",pod="pod_a"} 1000 2000
 				container_memory_working_set_bytes{container="container_a",namespace="namespace_b",pod="pod_b"} 1000 2000
 				container_memory_working_set_bytes{container="container_b",namespace="namespace_a",pod="pod_a"} 1000 2000
+				# HELP container_start_time [ALPHA] Start time of container in UNIX time
+				# TYPE container_start_time gauge
+				container_start_time{container="container_a",namespace="namespace_a",pod="pod_a"} 1e+09
+				container_start_time{container="container_a",namespace="namespace_b",pod="pod_b"} 1e+09
+				container_start_time{container="container_b",namespace="namespace_a",pod="pod_a"} 1e+09
```

Comment:
I think if it is a floating point number, the precision will be lost. Can you add a case to verify this?